followthemoney/schema/Interval.yaml

```diff
@@ -15,6 +15,9 @@ Interval:
     summary:
       label: "Summary"
       type: text
+    notes:
```

Comment:
Do we really need another "misc" field? We've got like 4 of them on `Thing` and it's a total mess :/